lxd/device/tpm.go

```diff
@@ -38,8 +38,10 @@ func (d *tpm) validateConfig(instConf instance.ConfigReader) error {
 
 	if instConf.Type() == instancetype.Container {
 		rules["path"] = validate.IsNotEmpty
+		rules["pathrm"] = validate.IsNotEmpty
 	} else {
 		rules["path"] = validate.Optional(validate.IsNotEmpty)
+		rules["pathrm"] = validate.Optional(validate.IsNotEmpty)
```

Comment:
What is "rm" and can it always be derived from path? Do we need a new setting for it?